### ISSUE TITLE
MODEL_COL_FIELD binding replacement broken after 3.2.6

### ISSUE THREAD

**Issue by goldenram** [angular-ui/ui-grid#5823]
In` ui-grid 3.2.6 and before this worked fine:

`<div class='ui-grid-cell-contents" title='{{MODEL_COL_FIELD}}'>{{COL_FIELD}}</div>`

The result was:

`<div class='ui-grid-cell-contents" title='row.entity.colName'>0.000</div>`

After 3.2.6 MODEL_COL_FIELD is getting incorrectly replaced with invalid code:

`<div class="ui-grid-cell-contents text-right" title="{{row[" entity']['pipemv']}}'="">0.000</div>`

Notice the double quote that is incorrect in the above, which causes the title to display as "{{row[

Basically, inside a cell template, I want to get the name, not value, of the column that I am working with.  If this has been updated, please let me know a better way to access this.

**Comment by mportuga** [comment]
Cannot reproduce:
http://plnkr.co/edit/6ImnsR2PV23dK9oCoYqB?p=preview
